crates/runtime/Cargo.toml

```diff
@@ -134,11 +134,12 @@ azure_storage_blobs = "0.21.0"
 bollard = "0.18.0"
 flightrepl = { path = "../flightrepl" }
 insta.workspace = true
+jsonpath = "0.1.1"
```

Comment:
Why is this required as a dependency of runtime?

Reply:
Bad merge conflict resolution when this branch was based atop https://github.com/spiceai/spiceai/pull/3576. Will remove.

Reply:
It was the package used before `jsonpath_rust`